tests/antool/decompress-bin-arch.sh

```diff
@@ -59,13 +59,26 @@ if [ "$FILES_ARCH" == "" ]; then
 fi
 
 echo -n "Decompressing vltrace's binary logs... "
+
 for file in $FILES_ARCH; do
 	bzip2 -d -f -k $file
 done
+
+TEMP_FILE=$(mktemp)
+
+# concatenate log files with syscall table
+for file in $SRC_DIR/$MASK_BIN; do
+	cp $file $TEMP_FILE
+	cp $SRC_DIR/$SYSCALL_TABLE $file
+	cat $TEMP_FILE >> $file
+done
```

Comment:
Commit message doesn't mention changes in shell scripts. I know what's going on, but...